airflow/models/baseoperator.py

```diff
@@ -1044,8 +1044,7 @@ def get_direct_relatives(self, upstream: bool = False) -> List[str]:
             return self.downstream_list
 
     def __repr__(self):
-        return "<Task({self.__class__.__name__}): {self.task_id}>".format(
-            self=self)
+        return f"<Task({self.__class__.__name__}): {self.task_id}>"
```

Comment:
I know this isn't your change, but there's no `Task` class in Airflow, so IMO it doesn't make sense to print "Task" here. Can we change to:

```suggestion
        return f"<{self.__class__.__name__}: {self.task_id}>"
```

Reply:
I agree with you that removing "Task" is a better way to go. Thanks for the suggestion